packages/react-components/src/components/gif.tsx

```diff
@@ -36,6 +38,7 @@ export type GifOverlayProps = {
 
 type GifProps = {
     gif: IGif
+    as?: 'a' | 'div'
```

Comment:
@thighhighcrocs not to overthink this but since we'd never pass `as="a"`, I almost feel like a boolean flag would be better than `as="div"`. I like `as`, because it's what we use with style components but It almost signifies that we could do something like `as="h1"`,  and you can't

Reply:
Ah yeah, that makes sense. What should we call the boolean? `noLink`?

Reply:
@thighhighcrocs that's exactly what I thought of too 👍